### PR TITLE
Docs for avg() method

### PR DESCRIPTION
**Pull request by fonini**

### CODE REVIEW

**Comment by HipsterJazzbo** [comment]
Ah you beat me to it :)

File: collections.md [hunk 1]
```diff
@@ -123,6 +124,26 @@ The `all` method simply returns the underlying array represented by the collecti
...
+<a name="method-avg"></a>
+#### `avg()` {#collection-method}
+
+The `avg` method returns the average of all items in the collection:
+
+    collect([1, 2, 3, 4, 5])->avg();
+
+    // 3
+
+If the collection contains nested arrays or objects, you should pass a key to use for determining which values to sum:
```
Comment (taylorotwell) [inline collections.md]: Which values to sum or which values to avg?

Comment (fonini) [inline collections.md]: Sorry, which values to avg(). Unfortunatelly, I can't fix it right now.
Em 26/09/2015 14:51, "Taylor Otwell" <notifications@github.com> escreveu:

**Comment by fonini** [comment]
Fixed.

**Pull request merged**